blog/2015/12/call-for-candidates-elections-2016.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+root: ../../..
+author: Jonah Duckles
+title: "Call for Candidates for the 2016 Steering Committee"
+date: 2015-12-08
+time: "06:00:00"
+category: ["Community", "Elections", "Steering Committee"]
+---
+
+<!-- start excerpt -->
+<p>From February 15-19, Software Carpentry will hold its annual election for the
+Steering Committee of The Software Carpentry Foundation. The inaugural Steering
+Committee has been an incredible team of dedicated community members working to
+position Software Carpentry so that it has a strong foundation and continued
+impact in scientific communities around the world. Please consider giving
+back to the community by standing for election!</p>
+<!-- end excerpt -->
+
+<p>If you are a qualified instructor who has taught at least twice in the past
+two years, or have done non-teaching work for the community, you can both stand
+for election and vote.</p>
+
+<p>In order to stand for election we request that
+you write a blog post that introduces yourself to the community. The post:</p>
+<ul>
+  <li>must be about 500 words </li>
+  <li>can be written in any format (question & answer, paragraph etc.),
+and</li>
+  <li> must be titled &quot;2016 Election: Your Name&quot;</li>
+</ul>
+
+<p>You can submit your post as a pull requestion to <a
+href="https://github.com/swcarpentry/site">this repository</a> or by email <a
+href="mailto:admin@software-carpentry.org">admin@software-carpentry.org</a>. In
+the post, you should explain:</p>
+<ul>
+  <li>your background how you've been involved in Software Carpentry</li>
+  <li>what you will do as a member of the Steering Committee to contribute to the growth and success of the community</li>
+</ul>
+
+<p>Blog posts must be submitted by Friday February 5th, one
+full week befor the election.</p>
```

Comment:
The timing (this announcement 69 days before the election if it lands today, and candidacy announcements 7 days before) is tighter than the current rules ([90 days and 30 days respectively][1]).  I'm personally fine with the smaller windows, since we're probably not going to have serious campaigning or anything, but there's not much point in having rules if we don't follow them ;).  The current rules also require [annual elections][1].  Since the last elections were in January (#705), I don't want to bump back the new election too far.  I'd suggest the steering committee vote in some new rules that allow a smaller window (30 day notice, and 7 day candidacy announcements?).

[1]: https://github.com/swcarpentry/board/blob/2d46b762a170b772ec04f2b7f0e91b51e2992fe2/governance.md#user-content-steering-committee-elections